e2e/tests/test_cases/single_replica_node_down.robot

```diff
@@ -0,0 +1,136 @@
+*** Settings ***
+Documentation    Single replica node down
+...              https://longhorn.github.io/longhorn-tests/manual/pre-release/node-not-ready/node-down/single-replica-node-down/
+
+Test Tags    manual_test_case
+
+Resource    ../keywords/common.resource
+Resource    ../keywords/deployment.resource
+Resource    ../keywords/persistentvolumeclaim.resource
+Resource    ../keywords/recurringjob.resource
+Resource    ../keywords/statefulset.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/workload.resource
+Resource    ../keywords/host.resource
+
+Library    OperatingSystem
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources include off nodes
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+
+*** Test Cases ***
+Single Replica Node Down Deletion Policy do-nothing With RWO Volume Replica Locate On Replica Node
+    Given Set setting node-down-pod-deletion-policy to do-nothing
+    When Create persistentvolumeclaim 0 using RWO volume
+    And Create deployment 0 with persistentvolumeclaim 0
+    And Write 100 MB data to file data in deployment 0
+
+    # Delete replicas to have the volume with its only replica located on different nodes.
+    And Update volume of deployment 0 replica count to 1
+    And Delete replica of deployment 0 volume on replica node
+    And Delete replica of deployment 0 volume on volume node
+    And Power off volume node of deployment 0
+    Then Wait for volume of deployment 0 attaching
+    And Wait for deployment 0 pods on volume node stuck in state Terminating
+
+    When Power on off node
+    And Wait for deployment 0 pods stable
+    And Wait for deployment 0 pod Running on replica node
+    Then Check deployment 0 data in file data is intact
+
+Single Replica Node Down Deletion Policy do-nothing With RWO Volume Replica Locate On Volume Node
+    Given Set setting node-down-pod-deletion-policy to do-nothing
+
+    When Create persistentvolumeclaim 0 using RWO volume
+    And Create deployment 0 with persistentvolumeclaim 0
+    And Write 100 MB data to file data in deployment 0
+
+    # Delete replicas to have the volume with its only replica located on the same node.
+    And Update volume of deployment 0 replica count to 1
+    And Delete replica of deployment 0 volume on all replica node
+    And Power off volume node of deployment 0
+    Then Wait for volume of deployment 0 faulted
+    And Wait for deployment 0 pods on volume node stuck in state Terminating
+
+    When Power on off node
+    And Wait for deployment 0 pods stable
+    And Wait for deployment 0 pod Running on volume node
```

Comment:
In this case, will the pod run on the previously powered off node? Or another node?

Volume node and replica node are dynamically changing depend on the volume is attached to which node and where are replicas of this volume located. 

Just try to clarify things here. Thank you!

Reply:
Yes, pod will on the origin(powered off) node. 
Updated keyword to `Check deployment 0 pod is Running on the original node`

[report.zip](https://github.com/longhorn/longhorn-tests/files/15318396/report.zip)